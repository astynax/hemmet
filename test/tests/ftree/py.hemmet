|py|src/*some-package/{*foo/ bar} test/*some-package/{*foo/ bar} README.md !test-cases/!test-case
