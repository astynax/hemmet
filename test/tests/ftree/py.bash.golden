#!/bin/bash
cat <<PREVIEW_END
This file tree will be created:
.
├── README.md
├── src/
│   └── some_package/
│       ├── __init__.py
│       ├── bar.py
│       └── foo/
│           └── __init__.py
├── test/
│   └── some_package/
│       ├── __init__.py
│       ├── bar.py
│       └── foo/
│           └── __init__.py
└── test-cases/
    └── test-case
PREVIEW_END
read -p "Press any key to continue..." -n1 -s
set -euf -o pipefail
touch "README.md"
mkdir "src" && pushd "src"
  mkdir "some_package" && pushd "some_package"
    touch "__init__.py"
    touch "bar.py"
    mkdir "foo" && pushd "foo"
      touch "__init__.py"
    popd
  popd
popd
mkdir "test" && pushd "test"
  mkdir "some_package" && pushd "some_package"
    touch "__init__.py"
    touch "bar.py"
    mkdir "foo" && pushd "foo"
      touch "__init__.py"
    popd
  popd
popd
mkdir "test-cases" && pushd "test-cases"
  touch "test-case"
popd
